qcodes/instrument/parameter.py

```diff
@@ -156,6 +156,12 @@ class _BaseParameter(Metadatable):
         snapshot_value (Optional[bool]): False prevents parameter value to be
             stored in the snapshot. Useful if the value is large.
 
+        snapshot_exclude (Optional[bool]): True prevents parameter to be
+            included in the snapshot. Useful if there are many of the same
+            parameter which are clogging up the snapshot.
+
```

Comment:
Please remove this blank line. I think this will render incorrectly in the html of the docs otherwise

Reply:
Done!